test/message-generator/test/job-declarator-flag-test/job-declarator-flag-test.json

```diff
@@ -0,0 +1,71 @@
+{
+    "version": "2",
+    "doc": [
+        "This test does",
+        "Launch the jd-server"
+    ],
+    "frame_builders": [
+    ],
+    "actions": [
+    ],
+    "setup_commands": [
+        {
+            "command": "cargo",
+            "args": [
+                        "llvm-cov",
+                        "--no-report",
+                        "run",
+                        "-p",
+                        "jd_server",
+                        "--",
+                        "-c",
+                        "../test/config/jds-do-not-fail-on-wrong-txdatasucc/jds-config.toml"
```

Comment:
hmm are you repurposing the config file from a different test?

I'd lean towards copying this directory into a new one with a more meaningful name. Even if the contents are identical and it ends up being somewhat redundant, if feels kinda dirty to have something here that is completely unrelated to the test.

Reply:
Okay, I will create a separate configuration file. It makes sense to have different configuration files for different contexts.